tests/Twig/Extension/SonataAdminExtensionTest.php

```diff
@@ -386,6 +386,64 @@ public function testRenderListElement(string $expected, string $type, $value, ar
         );
     }
 
+    /**
+     * @group legacy
```

Comment:
Please, add a `NEXT_MAJOR` comment explaining if the test or the `@group` annotation must be removed in the next major release.